### PR TITLE
fix template issues

### PR DESCRIPTION
**Pull request by devinea**
https://github.com/SAP/open-ux-tools/issues/1433

- Remove non-localised title in `view.xml` templates.
- Remove `iExecuteSearch` in V4 OPA test as Go button is no longer visible (need version check?)

### CODE REVIEW

File: packages/ui5-test-writer/templates/v4/integration/FirstJourney.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ sap.ui.define([
 <% if (startLR) { %>
             opaTest("Navigate to ObjectPage", function (Given, When, Then) {
                 // Note: this test will fail if the ListReport page doesn't show any data
+                <% if (!hideFilterBar) { %>
```
Comment (devinea) [inline packages/ui5-test-writer/templates/v4/integration/FirstJourney.js:20]: Only add add `iExecuteSearch ` when filterbar is visible!!

File: packages/ui5-test-writer/src/index.ts [hunk 2]
```diff
@@ -33,21 +33,28 @@ function readManifest(fs: Editor, basePath: string): Manifest {
  * @param manifest - the app descriptor of the app
  * @returns the application type. An exception is thrown if it can't be found or if it's not supported
```
[the remark this answers is not in the document]
Comment (devinea) [reply packages/ui5-test-writer/src/index.ts]: @IainSAP  Thanks . Fixed in https://github.com/SAP/open-ux-tools/pull/1432/commits/f647ea2c246987aee59f0de96de43405eb286aa9

**Review by IainSAP — APPROVED** [review]
Thanks @devinea 

- Tests updated
- Changes are clear

**Review by tobiasqueck — APPROVED** [review]
* title changes are clear and its change is properly reflected in the snapshots
* the fn `getAppTypeAndHideFilterBarFromManifest` is not ideal because as the name indicates it does two things that should be separate (1) getting something and (2) updating something
* I am ok with it since name and comment make clear what it does and it can be improved later
* changeset ok

**Comment by devinea** [comment]
Thanks @tobiasqueck @IainSAP

**Pull request merged**